Prace_domowe/Praca_domowa7/Jamrozy_Bartosz/pd7.ipynb

```diff
@@ -0,0 +1,926 @@
+{
```

Comment:
<p>Ta część jest zbędna. W zadaniu na detekcję outlierów są tylko kategorie inliers i outliers, przy czym wiadomo że w zbiorze treningowym są tylko inliers aby dało się zadanie rozwiązać a więc n_components=1. Z tego względu zadanie to było trochę nietypowe.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>